instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -9,28 +9,37 @@ module Instrumentation
     module AwsSdk
       # Generates Spans for all interactions with AwsSdk
       class Handler < Seahorse::Client::Handler
-        def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+        SQS_SEND_MESSAGE = 'SQS.SendMessage'
+        SQS_SEND_MESSAGE_BATCH = 'SQS.SendMessageBatch'
+        SQS_RECEIVE_MESSAGE = 'SQS.ReceiveMessage'
+        SNS_PUBLISH = 'SNS.Publish'
+
+        def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
           return super unless context
 
-          service_name = context.client.class.api.metadata['serviceId'] || context.client.class.to_s.split('::')[1]
+          service_name = context.client.class.to_s.split('::')[1]
```

Comment:
I'm confused, why are we dropping the check for `serviceId`?

Reply:
Re-added the check for `serviceId`.

It was part of adjustments for support old versions `v2.0.x`
But it see there are more issues, so it removed `v2.0.x` from supported versions https://github.com/open-telemetry/opentelemetry-ruby/pull/1026/commits/f007501b534d3afd5a166319160cc91d6acdd9e4